### PR TITLE
Feature CORS enabling

### PR DESCRIPTION
**Pull request by jfleck1**
Adds CORS response headers to all requests that are configurable in the properties files.

https://developer.mozilla.org/en-US/docs/Web/HTTP/Access_control_CORS

### CODE REVIEW

File: katharsis-spring/src/main/java/io/katharsis/spring/boot/KatharsisSpringBootProperties.java [hunk 1]
```diff
@@ -33,11 +37,45 @@ public void setPathPrefix(String pathPrefix) {
         this.pathPrefix = pathPrefix;
     }
 
-	public Long getDefaultPageLimit() {
-		return defaultPageLimit;
-	}
+    public Long getDefaultPageLimit() {
```
Comment (chb0github) [inline katharsis-spring/src/main/java/io/katharsis/spring/boot/KatharsisSpringBootProperties.java]: Could you cut down some boiler plate code and use lombok, please

**Comment by chb0github** [comment]
I am curious what this PR is for. Are you trying to integrate a UI and need those headers sent so the browser doesn't choke?

**Comment by Ramblurr** [comment]
Does this play nice with spring security?

File: katharsis-spring/src/main/java/io/katharsis/spring/boot/KatharsisSpringBootProperties.java (hunk 1 above)
Comment (chb0github) [inline katharsis-spring/src/main/java/io/katharsis/spring/boot/KatharsisSpringBootProperties.java]: I actually am working on lomboking the code.

**Comment by chb0github** [comment]
Basically, I am rejecting this PR -- stuffing CORS headers into katharsis is inappropriate. There are simple ways to test integrating with katharsis apps without compromising the security of the app as a whole. 

I can help you set it up, but not here

**Comment by jfleck1** [comment]
And with that, I'll stop contributing. Seems like you have it all figured out. Enjoy.

**Comment by chb0github** [comment]
Your choice. I would be happy to help give you insight on how to enable this for your environment. But, we can't just accept a PR because someone submits one. You would have been better off with an issue ticket first before writing any code.

**Pull request closed without merging**